BUILD.md

```diff
@@ -0,0 +1,170 @@
+# Build FreeOrion
+
+
+## Hardware and OS Requirements
+
+FreeOrion will compile on most recent Windows, Mac OS X and Linux operating
```

Comment:
I think with Windows and OSX we can be a bit more specific: Windows: Windows 7 (64bit) or later, Mac: OSX 10.10 or later.

Reply:
Is the 64 bit limitation for Windows in place?

Reply:
Left out the 64 bit remark until clarified.

Reply:
> Is the 64 bit limitation for Windows in place?

I'm not sure what you mean by that. There are no settings in the VS project that would specifically require the Windows OS of machine on which you build FO to be of the 64bit variant. The 64bit requirement is just a consequence of building FO requiring more than 4GB RAM.

You could probably argue that having a min RAM requirement of >4GB and requiring a 64bit OS is basically redundant...